src/strategies/MangroveOrder.sol

```diff
@@ -204,16 +208,12 @@ contract MangroveOrder is Forwarder, IOrderLogic {
     // POST (else)
     // * (NAT_USER+`res.bounty`, OUT_USER+`res.takerGot`, IN_USER-`res.takerGave`)
     // * (NAT_THIS, OUT_THIS, IN_THIS)
-    emit OrderSummary({
-      mangrove: MGV,
-      outbound_tkn: tko.outbound_tkn,
-      inbound_tkn: tko.inbound_tkn,
-      fillWants: tko.fillWants,
-      taker: msg.sender,
-      takerGot: res.takerGot,
-      takerGave: res.takerGave,
-      penalty: res.bounty
-    });
+    emit OrderSummary(
```

Comment:
emitting a data structure is going to make log parser's life complicated.
To avoid stack to deep just define:
```solidity
function logOrderData(TakerOrder memory tko, TakerOrderResult memory res) internal {
  emit OrderSummary(...);
}
```

Reply:
Okay, makes sense. Changed it